src/modules/service-catalog/submitServiceItemRequest.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Field } from "../ticket-fields";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
+
+export async function submitServiceItemRequest(
+  serviceCatalogItem: ServiceCatalogItem,
+  requestFields: Field[],
+  associatedLookupField: Field | null | undefined,
+  baseLocale: string
+): Promise<Response | void> {
+  if (!serviceCatalogItem) {
+    return;
+  }
+  const currentUserRequest = await fetch("/api/v2/users/me.json");
+  const currentUser =
+    currentUserRequest.ok && (await currentUserRequest.json());
```

Comment:
we should throw an error if we cannot get the csrf token

Reply:
Ok added